assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/clients/DataCiteClientSystemTest.java

```diff
@@ -275,16 +352,22 @@ private void stubUpdateMetadataResponse(String expectedPathForUpdatingMetadata)
                 .withBody(HTTP_RESPONSE_OK)));
     }
 
-    private void verifyCreateDoiResponse(Doi actual) {
-        verifyUpdateMetadataResponse(metadataPathPrefix + FORWARD_SLASH + actual.getPrefix());
-    }
+    private void stubCreateDoiResponse(DraftDoiDto expectedResponseBody) {
 
-    private void stubCreateDoiResponse(Doi expectedCreatedServerDoi) {
-        stubFor(post(urlEqualTo(metadataPathPrefix + FORWARD_SLASH + DEMO_PREFIX))
+        stubFor(post(urlEqualTo(DOIS_PATH_PREFIX))
             .withBasicAuth(EXAMPLE_MDS_USERNAME, EXAMPLE_MDS_PASSWORD)
```

Comment:
Prefix should be `EXAMPLE_DATACITE_` now since we are using it for both REST and MDS api.